src/Symfony/Component/DependencyInjection/Tests/ContainerTest.php

```diff
@@ -662,6 +662,13 @@ public function testAlias()
         $this->assertTrue($c->has('alias'));
         $this->assertSame($c->get('alias'), $c->get('bar'));
     }
+
+    public function testThatCloningIsNotSupported()
+    {
+        $class = new \ReflectionClass('Symfony\Component\DependencyInjection\Container');
+        $clone = $class->getMethod('__clone');
+        $this->assertTrue($clone->isPrivate());
```

Comment:
Actually, [isCloneable()](http://php.net/manual/en/reflectionclass.iscloneable.php) is available for PHP >= 5.4.